Tests/BSONTests/BSONObjectIDTests.swift

```diff
@@ -48,7 +41,7 @@ final class BSONObjectIDTests: BSONTestCase {
 
     func testFieldAccessors() throws {
         let oid = try BSONObjectID("FEEEEEEEFBBBBBBBBBFAAAAA")
-        expect(oid.timestamp).to(equal(0xFEEE_EEEE))
+        expect(oid.timestamp).to(equal(Date(timeIntervalSince1970: 0xFEEE_EEEE)))
```

Comment:
Now that I think about it, a minor improvement to this would be to construct the `Date` from a format string that corresponds to the number of seconds, rather than just initializing from the hex itself. 

From just looking at the date now, we can use the following:
- hex substring for date: `5F07445C`
- format string for date: "2020-07-09 12:22:52"

You'll need to create a `DateFormatter` like is used below and use `format.date` to construct the `Date` used as the "expected" one.

Reply:
Looks like you may need to set the timezone to UTC to ensure this test works. From the "Working with fixed format date representations" section of the [`DateFormatter` docs](https://developer.apple.com/documentation/foundation/dateformatter), you can do that with `format.timeZone = TimeZone(secondsFromGMT: 0)`.

Reply:
this should be fixed now